src/js/control-bar/fullscreen-toggle.js

```diff
@@ -74,8 +73,13 @@ class FullscreenToggle extends Button {
     } else {
       this.player_.exitFullscreen();
     }
+    /**
+     * On IE11 and Edge, focus is moved to the window when fullscreen is
+     * enabled. This ensures screen readers keep focus on the FullscreenToggle
+     * element after activating the button.
+     */
+    this.el().focus();
```

Comment:
I'm wondering if this is going to work consistently; isn't the switch to Full Screen a *request* that may happen asynchronously? In that case, this focus() may happen before the request has completed.